yetanotherco/aligned_layer
# feat: add withdraw method for batchers in servicemanager

# Description

- Adds withdraw method for batchers in AlignedLayerServiceManager.

# To Test

- First, fund the batcher in the AlignedLayerServiceManager:
```
cast send <aligned_layer_service_manager_addr> --value 1ether --private-key <batcher_private_key>
```
- Then, check the funds:
```
cast call <aligned_layer_service_manager_addr> "balanceOf(address)(uint256)" <batcher_addr>
```
- Then, withdraw the funds:
```
cast send <aligned_layer_service_manager_addr> "withdraw(uint256)" 0.5ether --private-key <batcher_private_key>
```
- Lastly, check the funds again:
```
cast call <aligned_layer_service_manager_addr> "balanceOf(address)(uint256)" <batcher_addr>
```
